libraries/AP_HAL/utility/RCOutput_Tap.cpp

```diff
@@ -318,32 +313,30 @@ void RCOutput_Tap::_uart_close()
 bool RCOutput_Tap::_uart_open()
 {
     // open uart
-    _uart_fd = open(UART_DEVICE_PATH, O_RDWR | O_NOCTTY | O_NONBLOCK);
+    _uart_fd = open(HAL_RCOUTPUT_TAP_DEVICE, O_RDWR | O_NOCTTY | O_NONBLOCK);
     int termios_state = -1;
 
     if (_uart_fd < 0) {
-        PX4_ERR("failed to open uart device!");
+        ::fprintf(stderr, "failed to open uart device! %s\n", HAL_RCOUTPUT_TAP_DEVICE);
```

Comment:
Could you expand on why? On Linux it's actually where errors like this should be going rather than clutter the output.